hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/RecordListBuilder.java

```diff
@@ -376,7 +383,31 @@ public void revertChildrenOf(@NonNull final SingleTransactionRecordBuilderImpl r
                 followingChildRemoved = true;
             } else {
                 if (child.reversingBehavior() == ReversingBehavior.REVERSIBLE && SUCCESSES.contains(child.status())) {
+                    // clear "side effect" fields
+                    child.accountID(AccountID.DEFAULT);
```

Comment:
We want to use `null` for missing fields here, since the serialized protobuf will be different than just setting `AccountID.DEFAULT`.

Reply:
@tinker-michaelj got it, updated this one.